solution/text-extractor/text_extractor.py

```diff
@@ -42,55 +42,53 @@ def handler(event: dict, context: dict) -> dict:
         token = config["token"]
     except KeyError:
         return lambda_response(400, "You must include 'id', 'uri', and 'post_url' in the request body.")
+    with TemporaryDirectory() as temp_dir:
```

Comment:
this directory creates a temporary space for the file to be stored.  We use this to pass it around.  It exist in all functions within the with statement.

Reply:
Code below doesnt really change its just tabbed over.  The  only difference is we pas in the location of the file we create.